cmake/Modules/OpmCompile.cmake

```diff
@@ -13,11 +13,13 @@ macro (opm_compile opm)
   # all public header files are together with the source. prepend our own
   # source path to the one of the dependencies so that our version of any
   # ambigious paths are used.
-  set (${opm}_INCLUDE_DIR "${PROJECT_SOURCE_DIR}")
-  set (${opm}_INCLUDE_DIRS ${${opm}_INCLUDE_DIR} ${${opm}_INCLUDE_DIRS})
+  # set (${opm}_INCLUDE_DIR "${PROJECT_SOURCE_DIR}")
+  # set (${opm}_INCLUDE_DIRS ${${opm}_INCLUDE_DIR} ${${opm}_INCLUDE_DIRS})
 
   # create this library, if there are any compilation units
-  include_directories (${${opm}_INCLUDE_DIRS})
+  # include_directories (${${opm}_INCLUDE_DIRS})
+  include_directories("${PROJECT_SOURCE_DIR}")
+  include_directories (SYSTEM ${${opm}_INCLUDE_DIRS})
```

Comment:
I am wondering whether this might not have introduced unwanted side effects. Quoting the documentation:
"
If the ``SYSTEM`` option is given, the compiler will be told the directories are meant as system include directories on some platforms. Signalling this setting might achieve effects such as the compiler skipping warnings, or these fixed-install system files not being considered in dependency calculations - see compiler docs."

I always loved the dependency calculation that requested recompilation if I changed DUNE.